packages/design-tokens/design-token-previews/utils.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
```

Comment:
As we have a utils folder, shouldn't it be inside it withe a more descriptive name?